packages/components/checkbox/src/Checkbox.test.tsx

```diff
@@ -195,115 +194,52 @@ describe('Checkbox', () => {
     expect(onCheckedChange).toHaveBeenCalledWith(['baseball', 'soccer'])
   })
 
-  it('should render group with a label', () => {
```

Comment:
I don't understand why all these tests are being modified/removed

Reply:
I don't get why this refactor is needed (tests and documentation) maybe I'm missing something but the issue was to improve the a11y of the component and most of the changes aren't related with it.

Reply:
Some of the changes here are related to how we documented and tested `FormField` usage. As discussed with you and Julien (if I remember correctly) we agreed to remove sections (within tests and documentation) that were too closely related to `FormField` implementation.
It could have been made within another PR I guess, if it's your point, but maybe now, after two weeks, we can leave it as is? :innocent:

Reply:
👌🏻